app/uk/gov/hmrc/alcoholdutyreturns/models/SubscriptionSummary.scala

```diff
@@ -27,7 +27,7 @@ object ApprovalStatus extends Enum[ApprovalStatus] with PlayJsonEnum[ApprovalSta
   case object Approved extends ApprovalStatus
   case object SmallCiderProducer extends ApprovalStatus
   case object Insolvent extends ApprovalStatus
-  case object DeRegistered extends ApprovalStatus
+  case object Deregistered extends ApprovalStatus
```

Comment:
These will need to be reverted alongside the account microservice changes

Reply:
Restored